src/Sarif.UnitTests/TestData/SarifVersionOneToCurrentVisitor/ExpectedOutputs/NotificationExceptionWithStack.sarif

```diff
@@ -1,6 +1,6 @@
 {
-  "$schema": "http://json.schemastore.org/sarif-2.0.0-csd.2.beta.2019-02-20",
-  "version": "2.0.0-csd.2.beta.2019-02-20",
+  "$schema": "http://json.schemastore.org/sarif-2.0.0-csd.2.beta.2019-04-03",
+  "version": "2.0.0-csd.2.beta.2019-04-03",
```

Comment:
> "version": "2.0.0-csd.2.beta.2019-04-03", [](start = 1, length = 42)

i rebaselined only this file to avoid making the szie of this PR huge. I needed this because our pre-release transformer will not support this conversion if we have baseAddress.